### PR TITLE
Add documentation in created objects

### PR DESCRIPTION
**Pull request by jiabaow**

### CODE REVIEW

File: fe1-web/model/network/method/message/data/lao/CreateLao.ts [hunk 1]
```diff
@@ -42,6 +43,10 @@ export class CreateLao implements MessageData {
     this.id = msg.id;
   }
 
+  /**
+   * Creates a Lao object from a goven object
```
Comment (pierluca) [inline fe1-web/model/network/method/message/data/lao/CreateLao.ts:47]: typo:
```suggestion
   * Creates a Lao object from a given object
```

**Review by pierluca — CHANGES_REQUESTED** [review]
Minor typo, otherwise LGTM

File: fe1-web/model/network/method/message/data/chirp/AddChirp.ts [hunk 2]
```diff
@@ -37,6 +38,10 @@ export class AddChirp implements MessageData {
     }
   }
 
+  /**
+   * Creates a AddChirp object from a given object
```
Comment (Xelowak) [inline fe1-web/model/network/method/message/data/chirp/AddChirp.ts:42]:
```suggestion
   * Creates an AddChirp object from a given object
```

File: fe1-web/model/network/method/message/data/election/EndElection.ts [hunk 3]
```diff
@@ -46,6 +47,10 @@ export class EndElection implements MessageData {
     this.election = msg.election;
   }
 
+  /**
+   * Creates a EndElection object from a given object
```
Comment (Xelowak) [inline fe1-web/model/network/method/message/data/election/EndElection.ts:51]:
```suggestion
   * Creates an EndElection object from a given object
```

File: fe1-web/model/network/method/message/data/election/SetupElection.ts [hunk 4]
```diff
@@ -106,6 +107,10 @@ export class SetupElection implements MessageData {
     });
   }
 
+  /**
+   * Creates a SetupSection object from a given object
```
Comment (Xelowak) [inline fe1-web/model/network/method/message/data/election/SetupElection.ts:111]:
```suggestion
   * Creates a SetupElection object from a given object
```

File: fe1-web/model/network/method/message/data/lao/CreateLao.ts [hunk 5]
```diff
@@ -4,6 +4,7 @@ import { validateDataObject } from 'model/network/validation';
 import { ActionType, MessageData, ObjectType } from '../MessageData';
 import { checkTimestampStaleness, checkWitnesses } from '../Checker';
 
+/** Data sent to creat a Lao */
```
Comment (Xelowak) [inline fe1-web/model/network/method/message/data/lao/CreateLao.ts:7]:
```suggestion
/** Data sent to create a Lao */
```

File: fe1-web/model/network/method/message/data/lao/CreateLao.ts [hunk 6]
```diff
@@ -42,6 +43,10 @@ export class CreateLao implements MessageData {
     this.id = msg.id;
   }
 
+  /**
+   * Creates a Lao object from a given object
```
Comment (Xelowak) [inline fe1-web/model/network/method/message/data/lao/CreateLao.ts:47]:
```suggestion
   * Creates a CreateLao object from a given object
```

File: fe1-web/model/network/method/message/data/lao/UpdateLao.ts [hunk 7]
```diff
@@ -7,6 +7,7 @@ import { validateDataObject } from 'model/network/validation';
 import { ActionType, MessageData, ObjectType } from '../MessageData';
 import { checkTimestampStaleness, checkWitnesses } from '../Checker';
 
+/** data sent to update a Lao */
```
Comment (Xelowak) [inline fe1-web/model/network/method/message/data/lao/UpdateLao.ts:10]:
```suggestion
/** Data sent to update a Lao */
```

File: fe1-web/model/network/method/message/data/lao/UpdateLao.ts [hunk 8]
```diff
@@ -41,6 +42,10 @@ export class UpdateLao implements MessageData {
     this.id = msg.id;
   }
 
+  /**
+   * Creates a UpdateLao object from a given object
```
Comment (Xelowak) [inline fe1-web/model/network/method/message/data/lao/UpdateLao.ts:46]:
```suggestion
   * Creates an UpdateLao object from a given object
```

File: fe1-web/model/network/method/message/data/meeting/CreateMeeting.ts [hunk 9]
```diff
@@ -5,6 +5,7 @@ import { validateDataObject } from 'model/network/validation';
 import { ActionType, MessageData, ObjectType } from '../MessageData';
 import { checkTimestampStaleness } from '../Checker';
 
+/** Data sent to creat a Meeting event */
```
Comment (Xelowak) [inline fe1-web/model/network/method/message/data/meeting/CreateMeeting.ts:8]:
```suggestion
/** Data sent to create a Meeting event */
```

File: fe1-web/model/network/method/message/data/rollCall/CreateRollCall.ts [hunk 10]
```diff
@@ -80,6 +81,10 @@ export class CreateRollCall implements MessageData {
     this.id = msg.id;
   }
 
+  /**
+   * Creates a CreatRollCall object from a given object
```
Comment (Xelowak) [inline fe1-web/model/network/method/message/data/rollCall/CreateRollCall.ts:85]:
```suggestion
   * Creates a CreateRollCall object from a given object
```

File: fe1-web/model/network/method/message/data/rollCall/OpenRollCall.ts [hunk 11]
```diff
@@ -46,6 +47,10 @@ export class OpenRollCall implements MessageData {
     this.update_id = msg.update_id;
   }
 
+  /**
+   * Creates a OpenRollCall object from a given object
```
Comment (Xelowak) [inline fe1-web/model/network/method/message/data/rollCall/OpenRollCall.ts:51]:
```suggestion
   * Creates an OpenRollCall object from a given object
```

**Review by Xelowak — APPROVED** [review]
Thanks for the addition ! I made some suggestions about typos. I think there should not be any changes to package.json if you just added documentation though.

**Comment by jiabaow** [comment]
> Thanks for the addition ! I made some suggestions about typos. I think there should not be any changes to package.json if you just added documentation though.
Thanks for the suggestions! Some parts in package-lock have been updated after "npm i", I hope that's not a big problem.

**Review by pierluca — APPROVED** [review]
LGTM

**Pull request merged**